conduit/Data/Conduit/Internal/Conduit.hs

```diff
@@ -812,6 +813,29 @@ ConduitM left0 =$= ConduitM right0 = ConduitM $ \rest ->
   where
 {-# INLINE [1] (=$=) #-}
 
+fuseEither :: Monad m => ConduitM a b m u -> ConduitM b c m d -> ConduitM a c m (Either u d)
```

Comment:
There should be a Haddock comment on this function explaining its purpose and how it's different from `=$=`.